tests/test_indexer.py

```diff
@@ -471,7 +473,17 @@ def delete_subscription(self, subscription_id):
         url = str(UrlBuilder().set(path=f"/v1/subscriptions/{subscription_id}")
                   .add_query("replica", self.replica.name)
                   .add_query("subscription_type", "elasticsearch"))
-        self.assertDeleteResponse(url, requests.codes.ok, headers=get_auth_header())
+
+        # This is where we want to do a mock patch
+        # (how to get the deleting user?)
+        # get_auth_header -> jwt_service_token -> various ways of populating email claim
+        deleting_users = [
+            os.environ.get('DSS_GCP_SERVICE_ACCOUNT_NAME') + '@' + os.environ.get('DSS_AUTHORIZED_DOMAINS_TEST'),
```

Comment:
There's probably a cleaner way to get the travis test google service acct email name, but this was the quickest way to do it. The Config class might need some new methods to make this nicer.

Reply:
Addressed in #151